arangod/RocksDBEngine/RocksDBLogValue.cpp

```diff
@@ -153,8 +148,8 @@ RocksDBLogValue::RocksDBLogValue(RocksDBLogType type, uint64_t dbId,
   switch (type) {
     case RocksDBLogType::CollectionCreate:
     case RocksDBLogType::CollectionChange:
-    case RocksDBLogType::CollectionDrop:
```

Comment:
is it intentional that the `CollectionDrop` case is removed here?

Reply:
yes